ratis-proto/src/main/proto/Raft.proto

```diff
@@ -280,6 +280,19 @@ message WatchRequestTypeProto {
   ReplicationLevel replication = 2;
 }
 
+message RoutePairProto {
+  RaftPeerProto from = 1;
+  RaftPeerProto to = 2;
+}
+
+message DataStreamRouteTableProto {
+  repeated RoutePairProto routeTable = 1;
+}
```

Comment:
Let's call the RouteProto and RouteTableProto.  It may be useful outside DataStream in the future.

Let's use RaftPeerId instead of RaftPeerProto.  It may be a good time to create RaftPeerIdProto which is currently missing.